src/Entity/RulesComponent.php

```diff
@@ -142,6 +154,20 @@ public function label() {
   }
 
   /**
+   * Returns the description.
+   */
+  public function getDescription() {
+    return $this->description;
+  }
+
+  /**
+   * Returns the tag.
+   */
+  public function getTag() {
```

Comment:
We actually have multiple tags. I think that's broken right now, but let's do already a proper method names -> getTags() which returns string[].